src/components/Garden/Feed/WrapToken.js

```diff
@@ -113,7 +110,10 @@ function Token({ balance, loading, mode, onClick, token }) {
         ${textStyle('body2')};
       `}
     >
-      <img src={icon} height="48" width="48" />
+      {apy === '0.00%' || !apy ? 
+       (<img src={icon} height="48" width="48" />) 
+       : (<span><img src={icon} height="48" width="48" css={`vertical-align: middle`} /> {apy} APY</span>)
+      }
```

Comment:
instead of doing that we can use the styled components defined before: 

```suggestion
{
        <TokenHeaderDiv><img src={icon} height="48" width="48" /> {apy && <ApySpan >{apy}% APY</ApySpan>}</TokenHeaderDiv>
      }
```